src/Parser/Mage/Fire/Modules/Features/Checklist.js

```diff
@@ -0,0 +1,178 @@
+      },
+    }),
+    new Rule({
+      name: 'Managing  your procs',
+      description: <Wrapper>Fire Mage is heavily dependent on correctly using your <SpellLink id={SPELLS.FIRE_BLAST.id}/> and <SpellLink id={SPELLS.PHOENIXS_FLAMES.id}/> guaranteed crit abilities to properly convert <SpellLink id={SPELLS.HEATING_UP.id}/> to <SpellLink id={SPELLS.HOT_STREAK.id}/>. Remember to only use your guaranteed crit spells to convert Heating Up to Hot Streak, and also remember to precede each instant cast <SpellLink id={SPELLS.PYROBLAST.id}/> with a hardcast of <SpellLink id={SPELLS.FIREBALL.id}/>, <SpellLink id={SPELLS.SCORCH.id}/>, or <SpellLink id={SPELLS.PYROBLAST.id}/> (If you have <ItemLink id={ITEMS.MARQUEE_BINDINGS_OF_THE_SUN_KING.id}/>) to increase the chances of getting a new Heating Up or Hot Streak</Wrapper>,
+      requirements: () => {
+        return [
+          new Requirement({
+            name: "Fire Blast used without Heating Up",
+            check: () => this.heatingUp.fireBlastUtilSuggestionThresholds,
+          }),
+          new Requirement({
+            name: "Phoenix's Flames used without Heating Up",
+            check: () => this.heatingUp.phoenixFlamesUtilSuggestionThresholds,
+          }),
+          new Requirement({
+            name: "Hard Cast Before Hot Streak Util",
+            check: () => this.hotStreak.castBeforeHotStreakThresholds,
+          }),
+          new Requirement({
+            name: "Expired Hot Streaks",
+            check: () => this.hotStreak.expiredProcsThresholds,
+          }),
+          new Requirement({
+            name: "Wasted Crits",
+            check: () => this.hotStreak.wastedCritsThresholds,
+          }),
+        ];
+      },
+    }),
+    new Rule({
+      name: 'Managing Combustion',
+      description: <Wrapper><SpellLink id={SPELLS.COMBUSTION.id}/> is the only major cooldown that Fire Mage has and it plays a massive role in your performance. When Combustion is about to come off cooldown, it is important that you bank a couple charges of <SpellLink id={SPELLS.FIRE_BLAST.id}/> and <SpellLink id={SPELLS.PHOENIXS_FLAMES.id}/> to help you get as many Hot Streak procs as possible during Combustion. Additionally, during Combustion, your goal is to use your instant cast spells to convert Heating Up to Hot Streak as quickly as possible so you can get more instant cast Pyroblasts in during the Combustion Window. You should only hard cast spells like <SpellLink id={SPELLS.FIREBALL.id}/> and <SpellLink id={SPELLS.SCORCH.id}/> if you are completely out of instant cast spells. Finally, if you have <ItemLink id={ITEMS.MARQUEE_BINDINGS_OF_THE_SUN_KING.id}/>, you should make sure you are using your procs to hard cast <SpellLink id={SPELLS.PYROBLAST.id}/> during Combustion as long as you can finish the cast before Combustion ends.</Wrapper>,
+      requirements: () => {
+        return [
+          new Requirement({
+            name: "Combustion Used with < 2 Phoenix Flames Charges",
```

Comment:
If possible, I think it would make more sense to show how many PFs were actually cast per combustion, rather than the number available at the start of it.

Reply:
The point of that module is to make sure the player is banking charges so that they have them during Combustion. The number of uses during Combustion doesnt particularly matter because one Combustion could involve hard casting Pyroblast 3 times because they got lucky on Bracer Procs, and the next might be entirely full of instant casts because they got no Bracer Procs.

But the goal of Combustion is (with the exception of Bracer Procs) to get as many instant cast pyroblasts as possible, so if a player casts Combustion and they have 0 PF Charges and 1 Fire Blast charge ... that combustion is going to suck for them cause its going to be almost entirely full of hard casts.

So i could add the stat for the number used during combustion .. though idk that it would actually be meaningful info for the player. 

Same for Fire Blast.

Reply:
Oh ok, yeah I wasn't thinking about the Bracers. Sounds fine to leave as is then.